specification/requirements.adoc

```diff
@@ -63,15 +64,24 @@ As a consequence, a CoVE-IO supported DSM must support the SPDM specification
 version 1.2 or above. It must also support sending and receiving SPDM Secured
 Messages as defined in the <<SecuredSPDM>> specification version 1.1 or above.
 
-TODO: List of required SPDM messages
+The CoVE-IO supported DSM must support SPDM responder capabilities: CERT_CAP,
+MEAS_CAP (10b), ENCRYPT_CAP, MAC_CAP, KEY_EX_CAP.
+
+The CoVE-IO supported DSM must support below algorithms:
+1. For BaseAsymAlgo, one or more of the following: TPM_ALG_RSASSA_3072,
+   TPM_ALG_ECDSA_ECC_NIST_P256, TPM_ALG_ECDSA_ECC_NIST_P384.
+2. For BaseHashAlgo, one or more of the following: TPM_ALG_SHA_256, TPM_ALG_SHA_384.
+3. For MeasurementHashAlgo, one or more of the following: TPM_ALG_SHA_256, TPM_ALG_SHA_384.
+4. For DHE Group, one or more of the following: secp256r1, secp384r1.
+5. For AEAD Cipher Suite, the following: AES-256-GCM with 16 Byte MAC.
```

Comment:
Thanks for taking care of this @jyao1 .
I only have a few nits that I sent here https://github.com/jyao1/riscv-ap-tee-io/pull/2. Feel free to squash my commit if you're ok with it.

Reply:
Thanks. I have merged and squashed.